lib/elixir/lib/map.ex

```diff
@@ -91,6 +91,11 @@ defmodule Map do
       iex> %{map | three: 3}
       ** (KeyError) key :three not found
 
+  The functions in this module that need to find a specific key work in logarithmic time.
+  This means that, the larger the map, the longer it will take to find the key.
```

Comment:
```suggestion
  This means that the time it takes to find keys grows as the map grows, but it's not directly proportional to the map size.
```

or something like this?